db/db.go

```diff
@@ -4,19 +4,25 @@ import (
 	"database/sql"
 	"fmt"
 	"strings"
+	"time"
 
+	"github.com/getAlby/lndhub.go/lib/service"
 	"github.com/uptrace/bun"
 	"github.com/uptrace/bun/dialect/pgdialect"
 	"github.com/uptrace/bun/driver/pgdriver"
 	"github.com/uptrace/bun/extra/bundebug"
 )
 
-func Open(dsn string) (*bun.DB, error) {
+func Open(config *service.Config) (*bun.DB, error) {
 	var db *bun.DB
+	dsn := config.DatabaseUri
 	switch {
 	case strings.HasPrefix(dsn, "postgres://") || strings.HasPrefix(dsn, "postgresql://") || strings.HasPrefix(dsn, "unix://"):
 		dbConn := sql.OpenDB(pgdriver.NewConnector(pgdriver.WithDSN(dsn)))
 		db = bun.NewDB(dbConn, pgdialect.New())
+		db.SetMaxOpenConns(config.DatabaseMaxConns)
+		db.SetMaxIdleConns(config.DatabaseMaxIdleConns)
+		db.SetConnMaxLifetime(time.Duration(config.DatabaseConnMaxLifetime) * time.Second)
```

Comment:
that should be `time.Minute` for 30 miniutes - and not seconds?

Reply:
I've updated the default to 1800 - I would prefer seconds as a default time unit.